web-services-app/src/test/java/edu/unc/lib/boxc/web/services/rest/modify/ExportCsvIT.java

```diff
@@ -793,6 +902,21 @@ private void assertCsvRecord(List<CSVRecord> csvList, ResourceType objType, PID
         fail("No CSV record with PID " + expectedPid.getId() + " present");
     }
 
+    private void assertCsvContentIsCorrect(List<CSVRecord> csvList, PID folderPid, PID workPid, PID filePid, Map<String, PID> pidList) {
```

Comment:
pidList isn't used here anymore so it can be removed

Reply:
thanks for catching!